src/axios-auth.js

```diff
@@ -0,0 +1,6 @@
+import axios from "axios";
+
+const instance = axios.create({
+    baseURL: 'http://localhost/'
```

Comment:
het is netter als je dit naar een .env verplaats, vooral als je het uiteindelijk wilt deployen scheelt dat veel werk